content/guides/incremental.md

```diff
@@ -0,0 +1,130 @@
+# Incremental Adoption & Reusable Components
+
+Since Imba compiles to readable JavaScript, including arbitrary
+Imba code in your existing project is very easy. Moreover, since
+Imba components are actual custom elements, you can use them with
+both plain HTML projects and most JavaScript UI framework
+projects (React, Vue, Svelte, etc.)
+
+Using components from imported Imba files is as simple as
+appending their names with `-tag`. You don't need to export the
+tags. For example, if you import an Imba file that has *only* the
+following contents:
+
+```imba
+tag marshmellow
+  <self> "yo!"
+```
+
+In React, Vue, and Svelte you can use the component this way:
+
+```js
+<marshmellow-tag/>
+```
+
+See the following sections for how to import Imba files depending on your configuration.
+
+## With Vite
+
+The best way to include Imba files in your existing non-Imba
+project is with [Vite](https://vitejs.dev/). If your existing
+project doesn't use Vite, try [#with-modules](#with-modules) or
+[#with-npm](#with-npm) instead.
+
+- Add `imba` and `vite-plugin-imba` to your existing project:
+	```bash
+	npm i -D imba vite-plugin-imba
+	```
+
+- In `vite.config.js` import the Imba plugin:
+
+	```js
+	import { imba } from 'vite-plugin-imba'
+	```
+
+- And add `imba()` to the list of plugins:
+
+	```js
+		plugins: [vue(),imba()],
+	```
+
+- In your app you can now import any Imba files with:
+
+	```js
+	import './filename.imba'
+	```
+
+## With Modules
+
+If you use eslint, this option may not work without disabling
+some rules; try [#with-npm](#with-npm) instead.
+
+- In your non-Imba project, run:
+
+	```bash
+	npx imba create --template component imba-project
+	cd imba-project
+	npm run build
+	```
+
+- Import the Imba project into your non-Imba project as a module via:
+
+	```js
+	import './imba-project'
+	```
+
+## With NPM
+
```

Comment:
I find this section a bit confusing. I don't think we should recommend people to use npm link. It is something an advanced user would use for fun and only in a local machine. What happens when pushing to github? collaborating with another developer?

Reply:
You're right, `npm install ./package` would be better.